macros/add_pass_through_columns.sql

```diff
@@ -0,0 +1,13 @@
+{% macro add_pass_through_columns(base_columns, pass_through_var) %}
+
+  {% if pass_through_var %}
+
+    {% for column in pass_through_var %}
+
+      {% do base_columns.append({ "name": column.name, "datatype": column.datatype, "alias": column.alias }) if column.alias else base_columns.append({ "name": column.name, "datatype": column.datatype}) %}
```

Comment:
are we removing the datatype attribute from this part? forgot where we left off

Reply:
Yeah I think it makes the most sense to remove it as it is not needed and adds unnecessary complexity. Removed, thanks!